frontend/.eslintrc

```diff
@@ -37,7 +37,8 @@
     "semi": ["error", "always"],
     "import/prefer-default-export": "off",
     "linebreak-style": "off",
-    "object-curly-newline": "off"
+    "object-curly-newline": "off",
+    "max-len": "off"
```

Comment:
change it from `"off"` to 
`["warning", { "code": 120, "ignoreUrls": true }] `- we use printWidth 120 in prettier so there should be no problem with that rule